### PR TITLE
Created MessageView component, adjusted message comp to be dynamic

### PR DESCRIPTION
**Pull request by AnnaBBee**
- Created Message View container which returns Message List
- Adjusted MessageList: 
a)added mapping through messages
b)adjusted handleSubmit function to display a new message
c) added messagesList array to the state
- Message adjustments: 
a) Added convertTime function to display correct time
b) adjusted display of messages styling and minor fixes.

### CODE REVIEW

File: my-app/src/containers/channelListPage.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import ChannelCard from '../components/ChannelCard'
+import ChannelCard from '../components/channelCard'
```
Comment (bmordan) [inline my-app/src/containers/channelListPage.js:2]: try to be consistent.

File: my-app/src/components/messageList.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <h1 className="white bg-blue w-100 sans-serif ma0 pa2 fixed pl3">Sky 1</h1>
+        <div className="sans-serif pt5 pb5 pl3 pr3 flex flex-column">
+          <p className="">Welcome to the Sky 1 chat</p>
+          {this.state.messagesList.map((messageInfo, index ) => {
+            return <Message key={index} messageInfo={messageInfo}/>
+          })}
+        </div>
+        <div>
+          <form className="w-100 flex sans-serif ma0 ba fixed" style={{bottom:0}} onSubmit={this.handleSubmit} >
+            <input className="bg-white w-80" type="text" placeholder="Write a message here" onChange={event => this.setState({newMessage: event.target.value})}/>
```
Comment (bmordan) [inline my-app/src/components/messageList.js]: no.
don't set state in the render function.
pull this into a class method and reference it `this.onChange`

File: my-app/src/components/messageList.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        myMessage: true,
+        message: 'here is my message'
+      }
+    ]
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    this.setState({messagesList: [...this.state.messagesList, {
+        userName: 'Caroline',
```
Comment (bmordan) [inline my-app/src/components/messageList.js:27]: messages you'll get from the server will be in this format:

```
"username|timestamp|messageText"
```

File: my-app/src/components/messageList.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      {
+      userName: 'Ralf',
+      timePosted: Date.now(),
+      myMessage: false,
+      message: 'here is someone\'s message'
+    },
+      {
+        userName: 'Caroline',
+        timePosted: Date.now(),
+        myMessage: true,
```
Comment (bmordan) [inline my-app/src/components/messageList.js:18]: how to tell if a message is yours?

you need the `username`

you'll have a string like this:

```
"someuser|1553699775343|hello you"
const [username, timestamp, message] = message.split('|')
```

username === username ? align left : align right

File: my-app/src/components/messageList.js (hunk 2 above)
Comment (AnnaBBee) [reply my-app/src/components/messageList.js]: fixed

File: my-app/src/components/messageList.js (hunk 4 above)
Comment (AnnaBBee) [reply my-app/src/components/messageList.js:18]: Leave it untill login is complited

**Review by bmordan — APPROVED** [review]

**Pull request merged**